Source/Packages/manifest.json

```diff
@@ -1,12 +1,42 @@
 {
   "dependencies": {
-    "com.unity.package-manager-ui": "2.0.3",
+    "com.unity.collab-proxy": "1.2.16",
```

Comment:
Are these now just added by Unity itself? Will they be automatically added by Unity? Should manifest.json be in .gitignore?

Example, below is com.unity.ide.vscode is that 1.1.3 your version? Or is this Unity explicitly saying they support this version?

Reply:
(by how I copy/paste the package, this manifest.json file won't matter).

Feel free to ignore this particular comment.

Reply:
I think these changes are necessary, and should be published. The project is invalid for anything but the Unity version that exactly matches this manifest.
Keeping this up to date with the latest version means at least that 1 version works, rather than nothing works.